tools/Environments/DevHome.Environments/ViewModels/LandingPageViewModel.cs

```diff
@@ -290,7 +290,7 @@ private async Task AddAllComputeSystemsFromAProvider(ComputeSystemsLoadedData da
                         packageFullName,
                         _windowEx);
                     await computeSystemViewModel.InitializeCardDataAsync();
-
+                    await computeSystemViewModel.InitializePinDataAsync();

```

Comment:
We could probably move the call into InitializeCardDataAsync(). What do you think? Up to you if you want to change

Reply:
Sure, probably makes more sense there